mdbook-tera-backend/src/tera_renderer/custom_component.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::Result;
+use serde::Deserialize;
+use std::path::{Path, PathBuf};
+use tera::Tera;
+
+/// Configuration in `book.toml` `[output.tera-renderer]`.
+#[derive(Deserialize)]
+pub struct TeraRendererConfig {
+    /// Relative path to the templates directory from the `book.toml` directory.
+    pub templates_dir: PathBuf,
```

Comment:
This should be `template_dir`, even though there are multiple templates in the directory (a bit like it's "shoe maker" and not "shoes maker" even though the person probably makes many shoes).